tests/unit/test_check_exposure_has_models.py

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
Do we need all attributes of exposures if we only want to test the `depends_on` key ?